descriptors/ModuleDescriptor-template.json

```diff
@@ -337,7 +337,8 @@
             "circulation.rules.loan-policy.get"
           ],
           "modulePermissions": [
-            "circulation-storage.circulation-rules.get"
+            "circulation-storage.circulation-rules.get",
+            "inventory-storage.locations.item.get"
```

Comment:
How is the check out session affected by the need to fetch locations within the circulation rules? Or is this a bug fix that is included here for convenience?

Reply:
Yes, it's the fix for a bug, that prevented me from checking that check-out endpoint has enough module permissions.